extension/runtime/src/main/java/org/iris_events/producer/EventProducer.java

```diff
@@ -195,6 +267,30 @@ private void doSend(final Object message, final String userId, final boolean pro
         }
     }
 
+    public void sendRpcResponse(final Object message, final String replyTo) {
```

Comment:
This should be private or package private method

Reply:
This is solved by refactoring a bit. No such functionality is now in the public api.